test/conftest.py

```diff
@@ -0,0 +1,28 @@
+# Nameko relies on eventlet
+# You should monkey patch the standard library as early as possible to avoid
+# importing anything before the patch is applied.
+# See http://eventlet.net/doc/patching.html#monkeypatching-the-standard-library
+import eventlet
+eventlet.monkey_patch()  # noqa (code before rest of imports)
+
+from nameko.containers import ServiceContainer
+import pytest
+
+
+@pytest.yield_fixture
+def container_factory():
```

Comment:
Might be worth moving this function to nameko/testing/utils. Almost every application defines this fixture